src/app/public/modules/date-range-picker/date-range.service.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  SkyLibResourcesService
+} from '@skyux/i18n';
+
+import {
+  BehaviorSubject
+} from 'rxjs/BehaviorSubject';
+
+import {
+  Observable
+} from 'rxjs/Observable';
+
+import 'rxjs/add/observable/forkJoin';
+
+import 'rxjs/add/operator/first';
```

Comment:
Shouldn't `rxjs/add/...` be above `rxjs/Observable...` and `rxjs/BehaviorSubject`?

Reply:
I've been putting imports that apply "side effects" directly after the imported type they're affecting:

1. Load `Observable` type.
2. Apply side effects of `forkJoin` to Observable type.

We can talk about this with the group.